lib/logstasher.rb

```diff
@@ -61,6 +63,9 @@ def log_as_json(payload, as_logstash_event: false)
     end
 
     def logger
+      # Return a /dev/null logger if logstasher is enabled or silencing was enabled.
+      return SILENT_LOGGER if !enabled? || silence_standard_logging?
```

Comment:
We shouldn't check silence_standard_logging? here, because that kills direct logging too.